<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qt.qrc">
    <normaloff>:/images/main.ico</normaloff>:/images/main.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuImport">
     <property name="title">
      <string>Import</string>
     </property>
     <addaction name="actionDirectory_As_Project"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
    </widget>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="actionClose_Project"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="menuImport"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionToolBar"/>
    <addaction name="actionStatusBar"/>
    <addaction name="separator"/>
    <addaction name="actionShow_File_Column"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_About"/>
    <addaction name="actionSuggestion"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="action_Replace"/>
    <addaction name="actionReplace_In_Files"/>
   </widget>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="action_Compile"/>
    <addaction name="action_Run"/>
    <addaction name="separator"/>
    <addaction name="action_Property"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Project"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_Project">
   <property name="text">
    <string>Close &amp;Project</string>
   </property>
  </action>
  <action name="actionClose_All">
   <property name="text">
    <string>&amp;Close All</string>
   </property>
  </action>
  <action name="action_Save_Project">
   <property name="text">
    <string>&amp;Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCo_mpile">
   <property name="text">
    <string>Co&amp;mpile...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F5</string>
   </property>
  </action>
  <action name="actionDecompile">
   <property name="text">
    <string>&amp;Decompile...</string>
   </property>
  </action>
  <action name="actionChm_Information">
   <property name="text">
    <string>Chm &amp;Information</string>
   </property>
  </action>
  <action name="actionPr_eference">
   <property name="text">
    <string>Pr&amp;eference</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ToolBar</string>
   </property>
  </action>
  <action name="actionStatusBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>StatusBar</string>
   </property>
  </action>
  <action name="actionCompiled_Help_File">
   <property name="text">
    <string>Compiled Help File...</string>
   </property>
  </action>
  <action name="actionHTML_Help_Message">
   <property name="text">
    <string>HTML Help Message</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Option">
   <property name="text">
    <string>&amp;Preference</string>
   </property>
  </action>
  <action name="action_Compile">
   <property name="text">
    <string>&amp;Compile</string>
   </property>
  </action>
  <action name="action_Build">
   <property name="text">
    <string>&amp;Build</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionPr_eference_2">
   <property name="text">
    <string>&amp;Find</string>
   </property>
  </action>
  <action name="action_Run">
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action_Property">
   <property name="text">
    <string>&amp;Properties</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_Import">
   <property name="text">
    <string>&amp;Import</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_Replace">
   <property name="text">
    <string>&amp;Find/Replace</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionClean">
   <property name="text">
    <string>Clean Project</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionReplace_In_Files">
   <property name="text">
    <string>Replace In Files</string>
   </property>
  </action>
  <action name="actionSuggestion">
   <property name="text">
    <string>Suggestion to us</string>
   </property>
  </action>
  <action name="actionDirectory_As_Project">
   <property name="text">
    <string>Directory As Project</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionShow_File_Column">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show File Column</string>
   </property>
  </action>
  <action name="actionTo_PDF">
   <property name="text">
    <string>To PDF</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qt.qrc"/>
 </resources>
 <connections/>
</ui>
